spec/zoneddatetime.html

```diff
@@ -1327,6 +1327,13 @@ <h1>
             1. Set _days_ to _days_ + _sign_.
           1. Else,
             1. Set _done_ to *true*.
+        1. If _days_ &ne; 0, then
```

Comment:
I do not see it is necessary to have the line
"1. If _days_ &ne; 0, then" here.